src/constants.ts

```diff
@@ -1,2 +1,16 @@
 // Common reference to the script identifier
 export const SCRIPT_ID = "data-react-paypal-script-id";
+export const DATA_CLIENT_TOKEN = "data-client-token";
+
+export const EMPTY_PROVIDER_CONTEXT_ERROR_MESSAGE =
+    "usePayPalScriptReducer must be used within a PayPalScriptProvider";
+export const EMPTY_PROVIDER_CONTEXT_CLIENT_TOKEN_ERROR_MESSAGE =
+    "A client token wasn't found in the provider parent component";
+
+const braintreeVersion = "3.79.1";
```

Comment:
A new version was recently released so let's use the latest:

```suggestion
const braintreeVersion = "3.81.0";
```